docs/get-started.md

```diff
@@ -1,6 +1,9 @@
 ## Get Started
 Use the following scripts to publish chart, enable connected cluster, and add OSM Arc extension.
 
+## Prerequisites
+- Running Kubernetes cluster
```

Comment:
Might be good to also mention this cluster should be created in a subscription and region that has the latest OSM arc extension available.

Reply:
I was under the impression that the Kubernetes cluster can be running anywhere. It is the arc cluster that has to be in the subscription and region that has the latest OSM arc extension available.

Reply:
> It is the arc cluster that has to be in the subscription and region

Technically, it's the region in which the connected cluster resource is created. Refer to https://docs.microsoft.com/en-us/azure/azure-arc/kubernetes/connect-cluster#create-a-resource-group

Reply:
I misunderstood your comment. Thank you.

Reply:
Not at all. Thanks for asking!

Reply:
added a line about the region in the environment variables section. Let me know if that looks good to you.

Reply:
Thanks for adding it! For private preview, only `eastus` and `westeurope` will be supported.

Reply:
oops, sorry the link you referenced says exactly that.